var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -323,6 +323,15 @@ def configure_args(self):
     def nvptx_install(self):
         spec = self.spec
         prefix = self.prefix
+        if 'isl' in spec:
```

Comment:
@lee218llnl would it be reasonable to create a `setup_environment` function (where you could use `prepend_path`)?

Also: normally I'd expect the compiler wrappers to add the `-L` entry to prevent this. Does it make sense to you why it would complain in spite of that?

Reply:
@scheibelp I moved the setting of LD_LIBRARY_PATH to setup_environment. The -L setting doesn't help since it is a run-time issue, not a link-time issue. Initially I thought this problem was bound to the build phase, but now it looks like I can't run the built gcc. Ultimately the cc1 executable fails with the libisl.so.19 loading error. It looks like rpaths aren't getting set in the executable. I will probe further, but this is now a WIP...

Reply:
I'm thinking this may be isolated to my configuration somehow. I may end up deleting PR.